crates/sui-core/src/unit_tests/move_package_upgrade_tests.rs

```diff
@@ -44,7 +44,7 @@ pub fn build_upgrade_test_modules(test_dir: &str) -> (Vec<u8>, Vec<Vec<u8>>) {
     path.extend(["src", "unit_tests", "data", "move_upgrade", test_dir]);
     let package = sui_framework::build_move_package(&path, build_config).unwrap();
     (
-        package.get_package_digest().to_vec(),
+        package.get_package_digest(true).to_vec(),
```

Comment:
I don't understand why this needs to calculate the digest with unpublished dependencies, could you explain?

Reply:
(although I see that it's consistent with `get_package_bytes` below it)

Reply:
Ah good spot! Looking at it, these should both be `false`. I was running into an issue long ago where I wasn't getting the set of modules out of `get_package_bytes` (because I was incorrectly setting the address in the package) so setting this to `true` solved that issue -- however that issue was resolved but forgot to change this back. Will fix :)